### PR TITLE
change: [M3-8622] - Update Images empty state as part of Image Service Gen2

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

- Updates the Images Landing page empty state based on UX's request as part of Image Service Gen 2 💾 

## Preview 📷

| Before  | After   |
| ------- | ------- |
| Screenshot 2024-09-23 at 11 25 37 AM | Screenshot 2024-09-23 at 11 25 02 AM |

## How to test 🧪

- Check new copy for correctness 👀 
- Verify all automated testing passes ✅ 

## As an Author I have considered 🤔

- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [x] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

**Review by dwiley-akamai — COMMENTED** [review]
Looks like we need to update `images-empty-landing-page.spec.ts` as well to reflect the new copy

File: packages/manager/src/features/Images/ImagesLanding/ImagesLandingEmptyStateData.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import type {
 
 export const headers: ResourcesHeaders = {
   description:
-    'Store your own custom Linux images, enabling you to rapidly deploy Linode Compute Instances preconfigured with the software and settings you require.',
+    'Store custom Linux images so you can rapidly deploy compute instances that are preconfigured with what you need.',
```
Comment (abailly-akamai) [inline packages/manager/src/features/Images/ImagesLanding/ImagesLandingEmptyStateData.ts:14]: am no english major but this is reads pretty bloated
maybe more succinct?

```suggestion
    'Store custom Linux images to rapidly deploy compute instances preconfigured with what you need.',
```

**Review by abailly-akamai — APPROVED** [review]
Approving pending e2e fix

File: packages/manager/src/features/Images/ImagesLanding/ImagesLandingEmptyStateData.ts (hunk 1 above)
Comment (bnussman-akamai) [reply packages/manager/src/features/Images/ImagesLanding/ImagesLandingEmptyStateData.ts]: Our Technical Writer was good with that change!

**Review by dwiley-akamai — APPROVED** [review]

**Pull request merged**